Components/Cards/cards.css

```diff
@@ -0,0 +1,45 @@
+/* cards */
+.cart-versions {
+    display: flex;
+    flex-wrap: wrap;
+}
+
+.cart {
+    margin: 1rem;
+    padding: 1rem;
+    max-width: 100%;
+    font-size: 10px;
+}
+
+.img-contents {
+    background: white;
+    width: 100%;
+    max-width: 220px;
+    color: black;
+}
+
+.img-size {
+    width: 100%;
+}
+
+.rs99 {
+    text-decoration: line-through;
+    color: gray;
+    font-size: 0.8em;
+}
+
+.off {
```

Comment:
Again .. you can rename this to `discount-text` or simply `text-orange` if you are using utility classes for colors